assembly/src/assembler/tests.rs

```diff
@@ -391,3 +392,44 @@ fn module_ordering_can_be_arbitrary() -> Result<(), Report> {
 
     Ok(())
 }
+
+#[test]
+fn can_assemble_a_multi_module_kernel() -> Result<(), Report> {
+    const KERNEL: &str = r#"
+        use.kernellib::helpers->h
+        export.foo
+            exec.h::get_caller
+        end"#;
+    const HELPERS: &str = r#"
+        export.get_caller
+            caller
+        end"#;
+    const PROGRAM: &str = r#"
+        begin
+            syscall.foo
+        end"#;
+
+    let context = TestContext::new();
+
+    let kernel_lib = {
+        let helpers = context.parse_module_with_path(
+            LibraryPath::new_from_components(
+                LibraryNamespace::User("kernellib".into()),
```

Comment:
I think this is the case, but just to confirm - we could have used just "kernel" (instead of "kernellib") here as well, right?

I'm mostly just curious because in `miden-base` we use "kernel" namespace (e.g., see [here](https://github.com/0xMiden/miden-base/blob/next/crates/miden-lib/asm/kernels/transaction/api.masm#L3)).